routes/project/location/kubernetes_cluster.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch(:project_location_prefix, "kubernetes-cluster") do |r|
+    r.web do
+      r.on NAME_OR_UBID do |kc_name, kc_ubid|
+        filter = if kc_name
+          {Sequel[:kubernetes_cluster][:name] => kc_name}
+        else
+          {Sequel[:kubernetes_cluster][:id] => UBID.to_uuid(kc_ubid)}
+        end
+
+        filter[:location] = @location
+        kc = @project.kubernetes_clusters_dataset.first(filter)
+
+        next 404 unless kc
+
+        r.get true do
+          authorize("KubernetesCluster:view", kc.id)
+
+          @kc = Serializers::KubernetesCluster.serialize(kc)
+          view "kubernetes-cluster/show"
+        end
+
+        r.delete true do
+          authorize("KubernetesCluster:delete", kc.id)
+          kc.incr_destroy
+          204
+        end
+
+        r.on "kubeconfig" do
+          r.get do
```

Comment:
```suggestion
        r.get "kubeconfig" do
```

With appropriate indentation fixes inside the block, and removal of extra `end`.  This isn't just for simplicity, the current code is incorrect as it will allow arbitrary segments after `kubeconfig` in the URL (since you did not pass an argument to `r.get`.

Reply:
Thanks, I changed accordingly